src/Request/ShowUser.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Request;
+
+class ShowUser
+{
+    private int $id;
+    private string $name;
+    private string $email;
+
+    public function __construct(int $id, string $name, string $email)
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->email = $email;
+    }
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function setId(int $id)
+    {
+        $this->id = $id;
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getEmail(): string
+    {
+        return $this->email;
+    }
+
+    public function setName(string $name)
```

Comment:
remove setters from this class. This should be readonly DTO.

Reply:
fixed